base/client.jl

```diff
@@ -211,6 +211,20 @@ end
 # call include() on a file, ignoring if not found
 include_ifexists(mod::Module, path::AbstractString) = isfile(path) && include(mod, path)
 
+function hascolorTTY()
+    term_type = get(ENV, "TERM", @static Sys.iswindows() ? "" : "dumb")
+    startswith(term_type, "xterm") && return true
+    try 
```

Comment:
There's trailing whitespace on this and several following lines, which is causing the whitespace buildbot check to fail.